docs/minutes/20201016.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: minutes
+date: 2020-10-16
+---
+
+### *v1.2 Postmortem*
+Fine$$e is coming into shape - Ian Yong, 2020  
```

Comment:
My previous comment https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/118#discussion_r506540945 was referring to Line 7, not Line 6